src/main/java/sirius/kernel/health/Log.java

```diff
@@ -131,86 +139,52 @@ public static synchronized Log get(String name) {
      * @param level  the desired log level
      */
     public static void setLevel(String logger, Level level) {
-        // Setup log4j
-        Logger.getLogger(logger).setLevel(level);
+        loggerHardReferences.computeIfAbsent(logger, Logger::getLogger).setLevel(level);
+    }
 
-        // Setup java.util.logging
-        java.util.logging.Logger.getLogger(logger).setLevel(convertLog4jLevel(level));
+    private void log(Level level, Object msg) {
+        StackTraceElement caller = new Throwable().getStackTrace()[2];
```

Comment:
Ist das sicher dass da keine NPE / IndexOutOfBounds auftritt?

Reply:
mostly yes - this is a private method, therefore it has to be called by another internal method. as this class hasn't a main method, this other method also has to have a caller.

one might be able to break this by forking a thread and passing in a curryed (curried?) method handle - but who would ever do this?